# Approach Flower Action

# goal definition
geometry_msgs/Pose flower_pose
geometry_msgs/Vector3Stamped flower_vector
---
# result definition
bool flower_approached
---
# feedback definition
manipulation_control/EECameraPose current_camera_pose
